### PR TITLE
refactor(Arrow.js): atom Arrow component cleanup

### PR DESCRIPTION
**Pull request by ASchmidt1024**
simple refactor of Arrow component incl. standard format

### CODE REVIEW

File: src/es/components/atoms/arrow/Arrow.js [hunk 1]
```diff
@@ -66,12 +71,69 @@ export default class Arrow extends Hover() {
...
+        0% { transform: translateY(0); }
+        48% { transform: translateY(-0.8em); opacity: 1; }
+        49% { transform: translateY(-0.8em); opacity: 0; }
+        50% { transform: translateY(0.8em); opacity: 0; }
+        51% { transform: translateY(0.8em); opacity: 1; }
+        100% { transform: translateY(0); }
+      }
+    `
+
+    let directionKeyframes = rightKeyframes
```
Comment (Weedshaker) [inline src/es/components/atoms/arrow/Arrow.js:120]: the directionKeyframes string should be applied to this.css = directionKeyframes after the switch, or did I miss something?

**Review by Weedshaker — COMMENTED** [review]
Were the arrow tests successful with all directions?

**Review by Weedshaker — COMMENTED** [review]
`constructor (...args) {
    super(...args)
  }`
this constructor could be removed

**Review by Weedshaker — APPROVED** [review]

**Review by Weedshaker — CHANGES_REQUESTED** [review]
pls test

File: src/es/components/atoms/arrow/Arrow.js (hunk 1 above)
Comment (ASchmidt1024) [reply src/es/components/atoms/arrow/Arrow.js:116]: 🙈  Jip, this is right and I forget it. Therefore, I put [more examples in Storybook for Arrow](https://github.com/mits-gossau/web-components-toolbox-storybook/commit/068fa8bc8dfb73fb58dca902c18fd1e4fbaa33a7). Muah!

**Review by Weedshaker — APPROVED** [review]

**Pull request merged**